src/loginServices.js

```diff
@@ -131,17 +152,17 @@ export function loadTranslations(store, locale, defaultTranslations = {}) {
 }
 
 /**
- * getLocale
- * return a promise that retrieves the tenant's locale-settings then
- * loads the translations and dispatches the timezone and currency.
- * @param {*} okapiUrl
- * @param {*} store
- * @param {*} tenant
+ * dispatchLocale
+ * Given a URL where locale, timezone, and currency information may be stored,
+ * request the data then dispatch appropriate actions for facet, if available.
  *
- * @return Promise
+ * @param {string} url location of locale information
+ * @param {object} store redux store
+ * @param {string} tenant
+ * @returns Promise
  */
-export function getLocale(okapiUrl, store, tenant) {
-  return fetch(`${okapiUrl}/configurations/entries?query=(module==ORG and configName==localeSettings)`,
+function dispatchLocale(url, store, tenant) {
+  return fetch(url,
```

Comment:
Should we move away from `fetch` towards something like `ky` in cases like this so we can get the headers for free? Esp now with i18n headers?

I know it's already on fetch and it'll be annoying to refactor. It's fine to say "naaaah, not now."

Reply:
Naaaah, not now. This is an easy/small refactor where I don't need to rewrite a bunch of tests. 

Good thinkin' though; filed [STCOR-546](https://issues.folio.org/browse/STCOR-546).